tests/test_zernike.py

```diff
@@ -349,6 +348,63 @@ def test_fit():
         np.testing.assert_allclose(resids2, 0, atol=1e-14)
 
 
+@timer
+def test_gradient():
+    # Start with a few that just quote the literature, e.g., Stephenson (2014).
+
+    Z11 = galsim.zernike.Zernike([0]*11+[1])
+
+    x = np.linspace(-1, 1, 100)
+    x, y = np.meshgrid(x, x)
+
+    def Z11_grad(x, y):
+        r2 = x**2 + y**2
+        gradx = 12*np.sqrt(5)*x*(2*r2-1)
+        grady = 12*np.sqrt(5)*y*(2*r2-1)
+        return gradx, grady
```

Comment:
This and the Z28 test below would benefit from some mathy comments.  E.g. write out the Noll definition of Z11, so we can read off that this is the right value of the derivative.